### PR TITLE
boards/weact-f4x1cx: enable tinyUSB

### PR DESCRIPTION
**Pull request by benpicco**
### Contribution description

The MCU already has tinyUSB support, we only need to mark it as available in the board, together with `periph_usbdev`. (Ideally this should just be a single USB port feature on which both `periph_usbdev` and `tinyusb_device` depend).

### CODE REVIEW

File: boards/common/weact-f4x1cx/Kconfig [hunk 1]
```diff
@@ -16,6 +16,7 @@ config BOARD_COMMON_WEACT_F4X1CX
     select HAS_PERIPH_TIMER
     select HAS_PERIPH_UART
     select HAS_PERIPH_USBDEV
+    select HAS_TINYUSB_DEVICE
```
Comment (gschorcht) [inline boards/common/weact-f4x1cx/Kconfig:19]: Kconfig won't be happy with this. This board and all the other boards with a signle USB interface was part of PR #18998 where I also changed the approach of the STDIO backend selection for Kconfig in commit https://github.com/RIOT-OS/RIOT/pull/18998/commits/b2c855afe282e39f57bc1fbd6f6295d4f8a75e41.

But with the current we woul have to use the STDIO backend selection as for `stm32f4discovery` or `esp32s-wemos-mini` dependent on what the default selection should be.

Comment (benpicco) [reply boards/common/weact-f4x1cx/Kconfig:19]: Ah true it's the problem again that would require a `weact-f4x1cx.conf` just to make Kconfig CI happy but without solving the issue on the Kconfig side.

**Comment by benpicco** [comment]
closed in favor of #18998

**Comment by gschorcht** [comment]
Why did you close it in favour of PR #18998? I don't think that PR #18998 will get merged as it is as long as the STDIO selection problem is not solved in Kconfig.

**Comment by benpicco** [comment]
I can always just set `CONTINUE_ON_EXPECTED_ERRORS=1`, that's preferable to dealing with another Kconfig issue :sweat_smile:

**Comment by gschorcht** [comment]
Why not to solve it for now like for `esp32s2_wemos_mini`?
```diff
       select HAS_HIGHLEVEL_STDIO
+     select MODULE_USBUS_CDC_ACM if TEST_KCONFIG && MODULE_USBUS
+     select PACKAGE_TINYUSB if TEST_KCONFIG && !MODULE_USBUS
+
+ choice STDIO_IMPLEMENTATION
+     default MODULE_STDIO_CDC_ACM if MODULE_USBUS
+     default MODULE_STDIO_TINYUSB_CDC_ACM if PACKAGE_TINYUSB
+ endchoice
```

**Comment by benpicco** [comment]
My only concern is that we will also have to clean up all those one-off selections once we have a general solution…

**Pull request closed without merging**